sdk/src/types/block/parent.rs

```diff
@@ -70,3 +73,7 @@ fn verify_parents<const VERIFY: bool>(parents: &[BlockId], _: &()) -> Result<(),
         Ok(())
     }
 }
+
+pub type StrongParents = Parents<1, 8>;
+pub type WeakParents = Parents<0, 8>;
+pub type ShallowLikeParents = Parents<0, 8>;
```

Comment:
I feel like those are likely to become wrapper types as their names seem to carry very different semantics, and this could become interesting for (at least) some metrics.

Reply:
Type alias here is very handy, I'll have a look if wrappers make sense 👍🏻

Reply:
I've come to the conclusion that it's not very desirable to have wrappers